Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            int age = Lesson3.Age(message);
+            return age;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
+            string wholename = Lesson3.WholeName(message); 
```

Comment:
it's called fullname, not wholename.

Reply:
Changed the value "wholename" to "fullname".